crates/aptos/src/move_tool/fmt.rs

```diff
@@ -26,28 +36,32 @@ pub enum EmitMode {
 
 #[derive(Debug, Args)]
 #[clap(group(clap::ArgGroup::new("input")
-.required(true)
+.required(false)
 .multiple(false)
-.args(&["file_path", "dir_path"]),
+.args(&["file_path", "dir_path", "package_path"]),
 ))]
 pub struct FmtCommand {
     /// How to generate and show the result after reformatting
     #[clap(long, value_enum, default_value = "overwrite")]
     emit_mode: EmitMode,
 
     /// Path to the file to be formatted
-    #[clap(long, group = "input")]
-    file_path: Option<PathBuf>,
+    #[clap(long, group = "input", hide = true)]
+    file_path: Option<PathBuf>, // this option is hidden
 
     /// Path to the directory to be formatted
     /// if neither file_path or dir_path is set, all move files in the current folder will be formatted
-    #[clap(long, group = "input")]
-    dir_path: Option<PathBuf>,
+    #[clap(long, group = "input", hide = true)]
+    dir_path: Option<PathBuf>, // this option is hidden
+
```

Comment:
I will add back the config path option. For `file-path` or `dir-path`, I don't think we want to provide these options to the user because they can always achieve the same thing by directly using the formatter binary themselves. For example, The command `cargo fmt` does not have these options available while the standalone tool `rustfmt` does.